assign_rights/models.py

```diff
@@ -93,7 +83,7 @@ def __str__(self):
 
 class Grouping(models.Model):
     title = models.CharField(max_length=255)
-    description = models.CharField(max_length=255)
+    description = models.TextField(max_length=255)
```

Comment:
Do you still want the `max_length` attribute?

https://docs.djangoproject.com/en/3.1/ref/models/fields/#textfield